src/main/kotlin/racingcar/RacingCarGame.kt

```diff
@@ -0,0 +1,26 @@
+package racingcar
+
+import racingcar.domain.Race
+import racingcar.domain.RaceSetting
+import racingcar.view.InputView
+import racingcar.view.ResultView
+
+class RacingCarGame {
+    companion object {
+        @JvmStatic
+        fun main(args: Array<String>) {
```

Comment:
java의 `public static void main(String[] args)` 같이 companion object로 만들 필요 없이, 최상위 함수로 main 함수를 바로 만들 수 있어요!  😃

```suggestion
class RacingCarGame {
    fun main() {
```

Reply:
README에 남기신 질문들은 관련된 코드 위치에 코멘트로 달게요! 😃


> main 함수는 테스트 코드를 작성하는 것이 맞을지?

A. 꼭 작성해야한다 혹은 작성할 필요 없다라고 단호하게 말할 수는 없을 것 같아요. 😅
하지만 가급적 테스트코드는 input에 대한 output이 명확한 비즈니스 로직에 작성하는 것이 중요하다고 생각하고 main 함수는 `InputView.getInput()`과 같이 콘솔로 입력을 받아야하는 케이스에 대해서 테스트 코드를 작성하는 것이 쉽지 않다고 생각해요. 🤔

> main 함수를 마치 controller와 같이 활용하고 있음 validation 및 race관련 함수 호출, 적절한 구조인지? 

A. 말씀하신 것처럼 main 함수가 Controller의 역할을 하고 있다고 느껴지네요. 😃 
물론 request 요청 값의 validation은 Controller에서도 진행할 수 있겠지만, 지금 수행하는 validation은 도메인 정책으로 녹일 수도 있을 것 같아요. 가급적 도메인 객체에 validation 로직을 두는 것이 좋다고 생각하지만, 지금의 경우에는 크게 잘못된 구조라고 생각하지는 않아요. 🙂

추후 5단계에서 MVC 패턴에 따라 코드를 `리팩터링`하게될텐데 이 때, 더 자세히 다뤄보면 좋을 것 같아요. 😃

Reply:
넵, 좋은 피드백 감사합니다. 마지막 step에서 리팩터링을 통해 좀 더 좋은 구조를 고민해보도록 하겠습니다 😄